Agents/APIAgent/README.md

```diff
@@ -101,7 +101,7 @@ Once this is done, the API Agent automatically manages the instantiation and all
 
 ##2.2 Extra Options
 ###2.2.1 API with dynamically generated URL
-It is common for external data API to use dynamically generated access URL. For this purpose, we define a new rdf property in `RMLTS.OWL`: `rmlts:hasDynamicTarget`. Find in [test_triples] folder an example to define an API source with dynamically generated URL: [api_temperature.ttl].
+It is common for external data API to use dynamically generated access URL. For this purpose, we define a new rdf property in `rmlts.owl`: `rmlts:hasDynamicTarget`. Find in [test_triples] folder an example to define an API source with dynamically generated URL: [api_temperature.ttl].
  
 ###2.2.2 API-to-TS mapping with Algebra
 Some source-data-to-target-Timeseries mapping may need algebra operations. For examples, the raw data contains 4 set of sub-category quantities and the target quantity is the sum of them. Such operations can be declared as a Named Individual of type `rmlts:Algebra`. And a corresponding implementation should be put in the [calculations module]. We have implemented one example: `rmlts:Sum`. An API meta-data example can be found in [test_triples] folder: [api_pvcapacity.ttl].
```

Comment:
Does the "Algebra" bit need updating?

Reply:
This has been addressed in the latest change commit.